src/protocols/rdp/guac_rdpdr/rdpdr_fs_service.c

```diff
@@ -153,5 +153,42 @@ void guac_rdpdr_register_fs(guac_rdpdrPlugin* rdpdr, char* drive_name) {
     /* Init data */
     device->data = rdp_client->filesystem;
 
+	/* Init directory for file transfer */
+	int create_fs = 0;
+
+	if (rdp_client) {
```

Comment:
Not sure this check is necessary, here - rdp_client is used previously in here under the assumption that it is not null, so this is probably not necessary.